{{cookiecutter.project_slug}}/.github/workflows/cypress.yml

```diff
@@ -66,4 +34,4 @@ jobs:
           NPM_CONFIG_PRODUCTION: false
           CYPRESS_TEST_USER_EMAIL: "cypress@example.com"
           CYPRESS_TEST_USER_PASS: {{ "${{ secrets.CYPRESS_TEST_USER_PASS }}" }}
-          CYPRESS_baseUrl: {{ "${{ needs.Setup.outputs.BASE_URL }}" }}
+          CYPRESS_baseUrl: {{ "${{ github.event.deployment_status.environment_url }}" }}
```

Comment:
All the comments I made above for the bootstrappers `cypress.yml` are relevant here as well

Reply:
buuut....since this file is even simpler now, we don't even need a `Setup` step anymore